modpack_format/modpack_format.md

```diff
@@ -0,0 +1,89 @@
+# MODIP Modpack Format
+
+The MODIP Modpack Format exists as a simpler alternative to the MODIP Index Format.
```

Comment:
Isn't it for a completely different purpose?

Reply:
Yeah, you're right, it's quite different.

Reply:
Resolved